src/containers/LogIn/style.scss

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.cdnfonts.com/css/vantage-brk');
+
+@import '../../styles/constants.scss';
+
+.login {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__box{
+    width: 500px;
+    background: $light_yellow;
+    padding: 15px;
+    border: 3px solid rgb(150, 150, 150);
+    box-shadow: 0px 0px 0px 1px rgb(90, 90, 90),
+      inset 0px 0px 0px 1px rgb(90, 90, 90),
+      1px 2px 0px 2px rgb(100, 100, 100);
+    border-radius: 10px;
+    align-items: center;
+    position: absolute;
```

Comment:
Even though designs make it so to use position: absolute, it's better to do this with flex